src/mocks/db.ts

```diff
@@ -16,12 +17,18 @@ export const mockMembers: Member[] = [
     name: 'current-member',
     email: '',
     extra: {},
+    type: 'individual',
```

Comment:
If you want there is also an enum for `MemberType`. This is optional.

Reply:
I'll change that in my next update. Thanks for the review.